### PR TITLE
XCTest simulator version of build_and_inject

### PR DESCRIPTION
**Pull request by nflore**
I was automating screenshots with XCTest and the normal version of the build_and_inject script didn't find the Clone simulator(s) that opens.

--set testing will show the simulators used there, added a separate file for it.

### CODE REVIEW

File: build_and_inject_test.sh
Comment (clindsay3) [inline build_and_inject_test.sh:1]: Maybe instead of a separate script this could be a flag that is passed to `build_and_inject.sh`, so we don't have to maintain two largely duplicate scripts?

File: README.md [hunk 1]
```diff
@@ -11,15 +11,18 @@ Modify the iOS Simulator so that it has a perfect status bar, then run your app
...
 
-## 1) Injecting into Springboard (Required on iOS 17+)
-**tl;dr** Running `build_and_inject.sh booted` will apply a default status bar to the running simulator. Replace "booted" with a simulator UDID to target a specific simulator.
+## Usage
+
+### 1) Injecting into Springboard (Required on iOS 17+)
+**tl;dr** Running `build_and_inject.sh booted` will apply a default status bar to the running simulator. Replace "booted" with a simulator UDID to target a specific simulator. Retrieve them with `xcrun simctl list devices | grep 'Booted'`
+
+**For XCTest simulators, use `build_and_inject_test.sh booted`** - Replace "booted" with a simulator UDID to target a specific simulator. Retrieve them with `xcrun simctl --set testing list devices | grep 'Booted'`
```
Comment (clindsay3) [inline README.md:19]: Nice, I think this is a great addition.

We should also clarify that this is only for duplicated simulators created by using the `-parallel-testing-enabled` flag on xcodebuild. If you are using XCTest but not using parallel testing, this line might be misleading.

**Review by clindsay3 — CHANGES_REQUESTED** [review]
Thanks so much for this update, this is great.

I actually use parallel testing, but took a different approach to solve this problem. I create a simulator, then run build_and_inject.sh on that simulator, then run my tests on that simulator using `xcodebuild -parallel-testing-enabled YES`. That way, the simulator is duplicated with the correct status bar already setup, and I have found that it maintains that across the duplication.

I'm not sure which approach is better, but in a way I wonder if that's simpler than needing to inject into each duplicated simulator individual? What do you think?

**Review by JPWhiteza — APPROVED** [review]

**Review by JPWhiteza — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -11,15 +11,18 @@ Modify the iOS Simulator so that it has a perfect status bar, then run your app
 
 Starting in Xcode 11, the `simctl` command line tool includes a `status_bar` option that allows you to override the appearance of the status bar in the simulator. Hopefully this will eventually supercede the need for SimulatorStatusMagic, but at the moment it still has holes that make this project continue to be relevant. In particular, `simctl status_bar` does not currently provide a way to add localized date and time strings in the status bar.
 
-## 1) Injecting into Springboard (Required on iOS 17+)
-**tl;dr** Running `build_and_inject.sh booted` will apply a default status bar to the running simulator. Replace "booted" with a simulator UDID to target a specific simulator.
```
Comment (JPWhiteza) [inline README.md:15]: ###

**Review by JPWhiteza — APPROVED** [review]

**Review by JPWhiteza — APPROVED** [review]

**Review by JPWhiteza — APPROVED** [review]
The request had been changed
